test-projects/complex-project/visivo_project.yml

```diff
@@ -0,0 +1,16 @@
+name: coffee_facts
+
+defaults:
+  target_name: snowflake
+
+includes:
+  - path: coffee_trade/coffee_trade.yml
```

Comment:
@tgsoverly I think this type of pattern will be a very good way for people to organize larger projects. It might be beneficial for us to create a `concepts` directory in the docs that sits next to `reference` and `background` where we have some articles and recommendations on how to use the tool